apiserver/facades/client/metricsdebug/metricsdebug.go

```diff
@@ -119,25 +120,41 @@ type byUnit []params.MetricResult
 func (t byUnit) Len() int      { return len(t) }
 func (t byUnit) Swap(i, j int) { t[i], t[j] = t[j], t[i] }
 func (t byUnit) Less(i, j int) bool {
+	if t[i].Unit == t[j].Unit {
+		if t[i].Key == t[j].Key {
+			return labelsKey(t[i].Labels) < labelsKey(t[j].Labels)
```

Comment:
This is a useful test quick test I don't see. Ensure you are sorting the way to expect, since you can have multiple labels, and those may be the only difference between otherwise duplicate units.